telemetry/definitions/commonDefinitions.json

```diff
@@ -1324,6 +1373,63 @@
         }
     ],
     "metrics": [
+        {
+            "name": "accessanalyzer_iamPolicyChecksCustomChecks",
+            "description": "Execution of Custom Policy Checks in IAM Policy Checks",
+            "metadata": [
+                {
+                    "type": "accessanalyzerCfnParameterFileUsed"
+                },
+                {
+                    "type": "accessanalyzerCheckType"
+                },
+                {
+                    "type": "accessanalyzerDocumentType"
+                },
+                {
+                    "type": "accessanalyzerFindingsCount"
+                },
+                {
+                    "type": "accessanalyzerInputPolicyType"
+                },
+                {
+                    "type": "accessanalyzerReferencePolicyType"
+                },
+                {
+                    "type": "result"
+                }
+            ]
+        },
+        {
+            "name": "accessanalyzer_iamPolicyChecksError",
+            "description": "Errors related to IamPolicyChecks",
+            "metadata": [
+                {
```

Comment:
why can we combine for Error with validation? 
result field can tell us if validation was successful for not. Also Error can have same metadata info as `accessanalyzer_iamPolicyChecksValidatePolicy` as reason, source, policy size, fileUsed, etc..

Reply:
We have some errors that are not always related to ValidatePolicy or CustomChecks. This would allow us to congregate all errors across the board, rather than looking at ValidatePolicy Failed, CustomChecks Failed, etc.

Reply:
Idea is to cover both cases of metric in one. Meaning `ValidatePolicy` will have both success & failed result, `CustomChecks` should cover success & failed result. If you feel the need to explicitly have Error metric, then we can discuss.

Reply:
Removed Error metric, using ValidatePolicy and CustomChecks to cover both results.

Reply:
looks good